code/modules/sanity/sanity_mob.dm

```diff
@@ -363,6 +363,43 @@
 					penalty *= -1
 				if(75 to 100)
 					penalty *= 0
+		//extra stuff if same departament
+		var/mob/living/carbon/human/fellow_human = M
```

Comment:
Maybe make human be passed as an argument then? And do `if(!istype(M)) return` instead of tabbing everything under `if(ishuman(M))`. I know you didn't make that proc, but might at least edit it without duct tape.

Reply:
good idea